mouse/record.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
```

Comment:
Drop this line as this shouldn't be an executable anymore. Also, make sure the executable bits have been removed when you commit this change (e.g. run `chmod -x mouse/record.py`).